main.js

```diff
@@ -0,0 +1,118 @@
+var addComment = function() {
+  var $this = $(this);
+  var $commentsContainer = $this.closest('.comments').find('.post-comments');
+  var $commentContentInput = $this.siblings('.comment-content');
+  var $commentAuthorInput = $this.siblings('.comment-author');
+  var content = $commentContentInput.val();
+  var author = $commentAuthorInput.val();
+
+  $commentsContainer.append(createComment(content, author));
```

Comment:
This is great - appending the result of `createComment` makes all of this real clean